src/components/Authentication/LoginPage.js

```diff
@@ -92,75 +122,317 @@ const LoginPage = () => {
+                                                <EmailOutlinedIcon style={{ color: "grey" }} />
+                                            </InputAdornment>
+                                        ),
+                                    }} />
+                                <PasswordInput autoComplete="current-password"
+                                    error={hasError || password === ""}
+                                    className="TextField"
+                                    inputProps={{ "data-cy": "passwordField" }} label="Password"
+                                    onChange={handleTextInput(setPassword)}
+                                    value={password} />
+                                <Grid container item className="optionsContainer">
+                                    <Grid item md={2} />
+                                    <Grid item md={4}>
+                                        <FormControlLabel
+                                            control={<Checkbox checked={shouldSave}
+                                                inputProps={{ "data-cy": "rememberMe" }}
+                                                onChange={toggleSavePassword} />}
+                                            label="Remember Me" />
+                                    </Grid>
+                                    <Grid item md={4} style={{ paddingTop: 10 }}>
+                                        <Link className="forgotPassword"
+                                            data-cy="forgotPassword" to={{
+                                                "pathname": "/forgotpassword",
+                                                "state": { email },
+                                            }}>
+                                            Forgot Password?
+                                        </Link>
+                                    </Grid>
+                                    <Grid item md={2} />
+                                    <Grid item md={4} />
+                                    <Grid item md={4} className="buttonSpacing">
+                                        <Button className="signInButton"
+                                            data-cy="signInButton"
+                                            type="submit" variant="contained">
+                                            SIGN IN
+                                        </Button>
+                                    </Grid>
+                                    <Grid item md={4} />
+                                </Grid>
+                            </Grid>
+                        </Grid>
+                    </form>
+                </div>)
+            default:
+                return (<div>default case here</div>)
+        }
+    }
+
+    const renderLogin = () => {
+        return (
+            userType ? renderOtherLogins() : renderEmailLogin()
+        );
+    }
+
+    return (
+        renderLogin()
```

Comment:
You don't need this function, just run the code: `userType ? renderOtherLogins() : renderEmailLogin()`

Reply:
@jsflca2012 please update this